src/features/DisplayControls/DrugChart/components/DrugChartView.jsx

```diff
@@ -36,7 +36,8 @@ const NoMedicationTaskMessage = (
 export default function DrugChartWrapper(props) {
   const { patientId } = props;
   const { config, isReadMode, visitSummary, visit } = useContext(IPDContext);
-  const { shiftDetails: shiftConfig = {}, drugChart = {} } = config;
+  const { shiftDetails: shiftConfig = {}, drugChart = {},enable24HourTime = {} } = config;
+  const enable24Hour = enable24HourTime;
```

Comment:
what's the use of defining the variable "enable24Hour"? we can directly use the variable "enable24HourTime"

Reply:
please do it for other files too